compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -292,7 +293,10 @@ object RefChecks {
             memberTp.overrides(otherTp,
                 member.matchNullaryLoosely || other.matchNullaryLoosely ||
                 warnOnMigration(overrideErrorMsg("no longer has compatible type"),
-                   (if (member.owner == clazz) member else clazz).srcPos))
+                   (if (member.owner == clazz) member else clazz).srcPos)) ||
+            // releaxed override check for explicit nulls
+            (ctx.explicitNulls && (member.is(JavaDefined) || other.is(JavaDefined)) &&
```

Comment:
See my comment in ProtoTypes.scala for how to avoid `stripAllNulls` here. 

In general, it would be great if we could at least avoid the deep `stripAllNulls` everywhere because that one is really expensive.

Reply:
`stripAllNulls` is no longer used